packages/firebase_ui_localizations/lib/src/get_localization.dart

```diff
@@ -11,6 +11,7 @@ final Set<String> kSupportedLanguages = {
   'de', // German
   'en', // English
   'es', // Spanish Castilian
+  'fi', // Finnish
```

Comment:
Finnish language needs adding to the switch statement here: https://github.com/firebase/FirebaseUI-Flutter/blob/main/packages/firebase_ui_localizations/lib/src/get_localization.dart#L57

Reply:
I made the addition and created new commit. Perhaps a mention of the necessary edits to all needed files could be added to Firebase UI Localizations README.